### ISSUE TITLE
What is the learning rate and how many iterations for train

### ISSUE THREAD

**Issue by lijialin03** [maziarraissi/DeepHPMs#4]
Hi, I noticed that N_iter = 0 for training in code. What is real number of N_iter?
# Train the identifier
    model.idn_u_train(N_iter=0)
    model.idn_f_train(N_iter=0)

What is more, there is no learning rate set for adam optimizer. Is that mean lr is 0.001 which is default?
# optimizer
    self.sol_optimizer_Adam = tf.train.AdamOptimizer()

Thank you for your reply!
